staff/groups/aleti/github-app-2.0.0-react/src/App.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import SearchUsers from './components/SearchUsers'
+import UsersList from './components/UsersList'
+import UserDetails from './components/UserDetails'
+import logic from './logic/index'
```

Comment:
TIP './logic' is enough, the import will automatically lookup the index.js

Reply:
👍